plugins/network/multibandwidth

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+: <<=cut
+
+=head1 NAME
+
+multibandwidth - Plugin to monitor the bandwidth between localhost and serveral hosts.
+
+=head1 APPLICABLE SYSTEMS
+
+This plugin is useful when you want to monitor the bandwidth between localhost and multiple hosts in a single graph or in several graphs making different soft links.
+
+=head1 REQUIREMENTS
+
+bing installed.
+
+You can install bing by using (Ubuntu/Debian): apt-get install bing
+
+=head1 CONFIGURATION
+
+The following is the default configuration
+
+[multibandwidth]
+user root
+env.hosts example.org example2.org example3.org
+env.samples 10
+env.small_packet_size 1000
+env.big_packet_size 8000
+
+- env.hosts explanation: hostname or IP of the hosts to calculate the bandwidth.
+
+- env.samples explanation: Reset stats after sending samples ECHO_REQUEST packets.
+
+- env.small_packet_size explanation: Specifies the number of data bytes to be sent in the small packets. The default and minimum value is 44.
+
+- env.big_packet_size explanation: Specifies the number of data bytes to be sent in the big packets. The default is 108. The size should be chosen so that big packet roundtrip times are long enough to be accurately measured.
+
+
+=head1 MAGIC MARKERS
+
+#%# capabilities=autoconf
+
+=head1 VERSION
+
+1.1.17
+
+=head1 AUTHOR
+
+Jose Manuel Febrer Cortés <https://www.linkedin.com/in/jfebrer/>
+Marco Bertola's help <https://www.linkedin.com/in/bertolamarco/>
+
+=head1 LICENSE
+
+GPLv2
+
+=cut
+
+
+
+case $1 in
+   config)
+                echo graph_title MultiBandwidth
+                echo 'graph_vlabel Mbps'
+                echo 'graph_args --base 1000 -l 0'
+                echo 'graph_scale no'
+                echo 'graph_category network'
+                echo 'graph_info This graph shows the bandwidth between localhost and serveral hosts'
+                for host in $hosts; do
+                        bandwidth=$((bandwidth+1))
+                        echo 'bandwidth'"$bandwidth"'.label ' "$host"
+                        echo 'bandwidth'"$bandwidth"'.draw LINE2'
+                        echo 'bandwidth'"$bandwidth"'.info Bandwidth statistics for '"$host"
+                done
+        exit 0;;
+    autoconf)
+                if hash bing 2>/dev/null; then
+                        echo 'yes'
+                        exit 0;
+                else
+                        echo 'no (bing not installed)'
+                        exit 0;
+                fi
+
+esac
+
+
+#Calculating the bandwidth
+for host in $hosts; do
+  bandwidth=$((bandwidth+1))
+  printf "%s""bandwidth""$bandwidth"".value ";
+  #bing output example: host:  4.592Mbps 0.223ms 0.217773us/bit
+  bing localhost "$host" -n -c 1 -e "$samples" -s "$small_packet_size" -S "$big_packet_size" 2>/dev/null \
+          | grep -E "bit$" \
+          | awk -F" +" '{ print $2}' | cut -c -5 \
```

Comment:
I assume, that you want to pick the second column (here: `4.592Mbps`)?
In this case `-F" +"` would be unnecessary, or?
Regarding the numeric format: I do not think, that it is safe to assume that you will always find `Mbps` here. Gigabit will surely arrive soon and kbps should also be no surprise.

How about something like the following?
```sh
awk '{print $2}' | sed 's/bps$//' | xargs numfmt --from auto
```
(without xargs you would need coreutils v8.25 - with xargs v8.21 is sufficient)
I think the additional requirement "numfmt" can reasonably be expected on most current systems (e.g. Debian Jessie - which will become `oldstable` in a few weeks or months).